unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWModeController.cs

```diff
@@ -68,19 +68,19 @@ public enum EditModeState
     private GameObject snapGO;
     private GameObject freeMovementGO;
 
-    public override void Init(BIWContext biwContext)
+    public override void Init(BIWContext context)
     {
-        base.Init(biwContext);
+        base.Init(context);
 
-        cursorGO = InitialSceneReferences.i.cursorCanvas;
-        cameraParentGO = InitialSceneReferences.i.cameraParent;
+        cursorGO = context.sceneReferences.cursorCanvas;
+        cameraParentGO = context.sceneReferences.cameraParent;
         InitGameObjects();
 
-        firstPersonMode = new BuilderInWorldFirstPersonMode();
-        godMode = new BuilderInWorldGodMode();
+        firstPersonMode = new BiwFirstPersonMode();
+        godMode = new BiwGodMode();
```

Comment:
We should normalize the `BIW` prefix. If its mean to be in ALL CAPS, please make it so on all classes that make use of it.